tracker/mtracker.py

```diff
@@ -1,5 +1,5 @@
 from flask import Flask,  json, request, jsonify
-from models import Request,requests, User, users
+from .models import Request,requests, User, users
```

Comment:
Remove the `.` on the models
You should have something like this
```python
from models import Request,requests, User, users
```
You can only do what you doing if the file that runs the application is out of the tracker module.

Reply:
In case you want to leave it like this. Create a file say `app.py` in the root folder of your application and use it to run your application i.e 
> In the file app.py
```python
from tracker.mtracker import app

if __name__=='__main__':
    app.run(debug=True)
```
> You can then run your application by running the command `py app.y`. If you do this, you have to remove the code below in the mtracker.py file
```python
if __name__=='__main__':
    app.run(debug=True)
```